### ISSUE TITLE
--key not passed to websockify when launched

### ISSUE THREAD

**Issue by jcaesar** [novnc/noVNC#1251]
**Describe the bug**
Using https fails and generates an obscure file not found error message.
```handler exception: [Errno 2] No such file or directory```

**To Reproduce**
1. Generate a keypair with `openssl req -x509 -nodes -days 42 -newkey rsa:1024 -keyout self.key -out self.pem`
2. Run the launch script `novnc  --cert $PWD/self.pem --ssl-only`
3. Hit port 6080 with a https request

**Expected behavior**
Things work.

**Client:**
 - Browser: httpie
 - Browser version: 1.0.2
 - OS: It really doesn't matter

**Server:**
 - noVNC version: 1.0.0
 - VNC server: x11vnc
 - WebSocket proxy: websockify 0.8.0

**Additional context**
The problem is that websockify is not passed a proper `--key` parameter. The obscure error message is likely caused by two facts:
 - websockify 0.8.0 defaults its key parameter to `''` and seems to do no further checks: https://github.com/novnc/websockify/blob/v0.8.0/websockify/websocket.py#L606
 - ssl.py won't do any checks either and straight up opens that file: https://github.com/python/cpython/blob/v3.8.0b1/Lib/ssl.py#L1374

This is backed at least by the fact that strace reports the attempt to open a file named `""`.

**Workaround**
Directly run websockify
```
websockify --web /usr/share/novnc --cert $PWD/self.pem --key $PWD/self.key 6080 localhost:5900 --ssl-only
```

**Comment by CendioOssman** [comment]
I assume you are referring to the `utils/launch.sh` script here. And indeed, it requires a combined cert/key file, as stated in its help output:

```
    --cert CERT           Path to combined cert/key file
                          Default: self.pem
```

But I see no problem adding flags to be able to split things, as websockify already supports that.

**Comment by CendioOssman** [comment]
Fixed in 97924ebd5d1d39c9752fa9642f15684a89a62e0b.
